lib/api/2.0/nodes.js

```diff
@@ -61,24 +62,45 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
```

Comment:
@brianparry it doesn't support AND queries? Just looking at the CQL docs (https://cassandra.apache.org/doc/cql/CQL.html) I'm finding examples like `SELECT ... WHERE KEY >= 'AM' and KEY =< 'AZ' AND module = 17;`. Though this definitely isn't my wheelhouse so I could be misunderstanding.

I think regardless I'd be definitely more in favor of a model change as you suggest to optimize queries like this, or perhaps creating new tables and/or collections that store these relationships so that we can make direct queries.